File: Ecole--Edtech1/src/ScolariteBundle/Entity/Coeff.php
<?php

namespace ScolariteBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Coeff
 *
 * @ORM\Table(name="coeff")
 * @ORM\Entity(repositoryClass="ScolariteBundle\Repository\CoeffRepository")
 *
 */
class Coeff
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var int
     *
     * @ORM\Column(name="valeur", type="integer")
     */
    private $valeur;

    /**
     * @return int
     */
    public function getValeur()
    {
        return $this->valeur;
    }

    /**
     * @param int $valeur
     */
    public function setValeur($valeur)
    {
        $this->valeur = $valeur;
    }

    /**
     * @var
     * @ORM\ManyToOne(targetEntity="Classe")
     * ORM\JoinColumn(name="niveau",referencedColumnName=Id)
     */
    private $niveau;

    /**
     * @var
     * @ORM\ManyToOne(targetEntity="Matiere")
     * ORM\JoinColumn(name="matiere",referencedColumnName=Id)
     */
    private $matiere;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param mixed $niveau
     * @return Coeff
     */
    public function setNiveau($niveau)
    {
        $this->niveau = $niveau;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getNiveau()
    {
        return $this->niveau;
    }

    /**
     * @param mixed $matiere
     * @return Coeff
     */
    public function setMatiere($matiere)
    {
        $this->matiere = $matiere;

        return $this;
    }

    /**
     * @return mixed
     */
    public function getMatiere()
    {
        return $this->matiere;
    }


}

File: ProjetPi/src/com/mycompany/myapp/entities/Coeff.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author dell
 */
public class Coeff {
   private int id;
   private int valeur;
   private Matiere matiere;
    private Classe niveau;

    public Coeff(int id, int valeur, Matiere matiere, Classe niveau) {
        this.id = id;
        this.valeur = valeur;
        this.matiere = matiere;
        this.niveau = niveau;
    }

    public Matiere getMatiere() {
        return matiere;
    }

    public void setMatiere(Matiere matiere) {
        this.matiere = matiere;
    }

    public Classe getNiveau() {
        return niveau;
    }

    public void setNiveau(Classe niveau) {
        this.niveau = niveau;
    }
    
    
    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getValeur() {
        return valeur;
    }

    public void setValeur(int valeur) {
        this.valeur = valeur;
    }

    
    
   

    

    public Coeff() {
    }

   
   
   
}
